test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -7615,6 +7615,13 @@ def forward(self, embedding_matrix, input, offset, weights):
         self.run_test(model, (embedding_matrix, x, offset, w))
 
     @skipIfUnsupportedMinOpsetVersion(11)
+    @skipIfUnsupportedMaxOpsetVersion(17)
+    @unittest.skip(
+        "This test is broken with ONNXRuntime(17): "
+        "when running with onnxruntime 1.17.0 this test fails with the following error:"
+        "FAIL : Non-zero status code returned while running If node. "
+        "Name:'/If' Status Message: if.cc:253 Compute "
+        "If nodes condition input must have exactly one element")
```

Comment:
@thiagocrepaldi need someone to take a look, whether it is export bug or regression in ort.

Reply:
Let's file a bug an fill in the xfail reason with it